cmake/modules/OmrDDRSupport.cmake

```diff
@@ -65,15 +65,23 @@ function(make_ddr_set set_name)
 		COMMAND "${CMAKE_COMMAND}" -E touch "${DDR_CONFIG_STAMP}"
 		WORKING_DIRECTORY "${DDR_BIN_DIR}"
 	)
+
+	# We need to set LIBPATH on z/OS so that ddrgen can find the a2e library
+	if(OMR_OS_ZOS)
+		set(DDRGEN_LIBPATH "LIBPATH=$<TARGET_FILE_DIR:j9a2e>:$LIBPATH")
+	else()
+		set(DDRGEN_LIBPATH)
+	endif()
+
```

Comment:
A better place for this would seem to be in `DDRSetStub.cmake.in` where `omr_ddrgen` is actually invoked.

In UMA builds, `LIBPATH` is also set on AIX, why is it not necessary there?

Reply:
> A better place for this would seem to be in DDRSetStub.cmake.in where omr_ddrgen is actually invoked.

I thought about that, but since its running in a separate instance of cmake we would still need something here to pass the relevant info  to the new cmake instnace (eg `-DJ9A2E_DIR=...`), so why not just set LIBPATH here

> In UMA builds, LIBPATH is also set on AIX, why is it not necessary there?
On the aix cmake builds we embed the libpath into the ddrgen executable https://github.com/eclipse/omr/blob/fb47dbd020bb3b9c2609111b927cfad8f485bb35/ddr/tools/ddrgen/CMakeLists.txt#L43

Reply:
Thanks for the pointer to the AIX solution.

I don't see why anything extra needs to be passed to that separate instance of cmake: won't the additions work more-or-less as is in `DDRSetStub.cmake.in`?

Reply:
the new instance of CMake does not have any definition of j9a2e target, so we can't use the `TARGET_FILE_DIR` generator expression.

Reply:
Ok, that's unfortunate.
```
set(DDRGEN_LIBPATH "LIBPATH=$<TARGET_FILE_DIR:j9a2e>:$LIBPATH")
```
should be
```
set(DDRGEN_LIBPATH "LIBPATH=$<TARGET_FILE_DIR:j9a2e>$${LIBPATH:+:$LIBPATH}")
```
so we don't add an empty entry to `LIBPATH` if it wasn't already there.